web/src/app/admin/admin-service-metrics/AdminServiceFilter.tsx

```diff
@@ -139,6 +139,7 @@ function AdminServiceFilter(): JSX.Element {
                     'site24x7',
                     'prometheusAlertmanager',
                     'email',
+                    'universal',
```

Comment:
should be conditional on the flag, we can do so by updating this component to use the types hook instead of hard-coding them

Reply:
https://github.com/target/goalert/pull/3837